cdc/processor.go

```diff
@@ -741,14 +754,14 @@ func (p *processor) addTable(ctx context.Context, tableID int64, replicaInfo *mo
 	ctx, cancel := context.WithCancel(ctx)
 	table := &tableInfo{
 		id:         tableID,
-		resolvedTS: replicaInfo.StartTs,
+		resolvedTs: replicaInfo.StartTs,
 		cancel:     cancel,
 	}
 	// TODO(leoppro) calculate the workload of this table
 	// We temporarily set the value to constant 1
 	table.workload = model.WorkloadInfo{Workload: 1}
 
-	startPuller := func(tableID model.TableID) {
+	startPuller := func(tableID model.TableID, isMarkTable bool) {
```

Comment:
how about this?
```suggestion
	startPuller := func(tableID model.TableID, resolvedTs *uint64) {
```
and we can remove `storeResolvedTs` function, just using `atomic.StoreUint64(resolvedTs, ts)`

Reply:
we can avoid having to judge in a loop by this way